### ISSUE TITLE
Error with minimalbedpe.sh

### ISSUE THREAD

**Issue by Sanaya30** [reskejak/ATAC-seq#8]
Hi,

I followed your workflow and am getting an error during peakcalling. 

This is the error I get from macs2: 
AssertionError: Right position must be larger than left position, check your BED file at line: b'131139202\tchr3\tchr3\t\n'

When I checked my treat.minimal.bedpe files, this is what I see- 
$ head treat.minimal.bedpe
131139202	chr3	chr3
30237118	chr2	chr2
84216081	chr15	chr15

My input tn5 file looks okay I think?
$ head treat.tn5.bedpe
chr3	131139188	131139234	chr3	131139202	131139247	NS500269:757:HMH7VBGX9:1:11101:1029:10932	44	+-
chr2	30236764	30236803	chr2	30237118	30237163	NS500269:757:HMH7VBGX9:1:11101:1032:2634	44	+-
chr15	84216030	84216076	chr15	84216081	84216126	NS500269:757:HMH7VBGX9:1:11101:1033:9687	44	+-

The command I used- 
bash bedpeMinimalConvert.sh treat.tn5.bedpe > treat.minimal.bedpe

macs2 callpeak -t treat.minimal.bedpe -f BEDPE -n treat -g mm --broad --broad-cutoff 0.05 --keep-dup all

Am I doing something wrong here? Please advice.

Best,
Sanaya

**Comment by reskejak** [comment]
Hi Sanaya,

Your output `treat.minimal.bedpe` file does not look correct.
It should look like:
```
chr3 131139188 131139247 NS500269:757:HMH7VBGX9:1:11101:1029:10932
```

Can you confirm that you are using a tab delimited file?

**Comment by Sanaya30** [comment]
Hi, 

Yes the file I am using is a tab delimited file. 

Since bedpeMinimalConvert.sh gave me error, I tried using this command instead

cut -f 1,2,6 treat.tn5.bedpe | sort -k1,1 -k2,2n -k3,3n > treat.minimal.bedpe

I get only the first three columns (as opposed to 4 from your code), but would essentially should be good to input to macs2. Do you think this would be correct?

Best,
Sanaya

**Comment by reskejak** [comment]
Yes, try using your resulting file with MACS2 and let me know if it works.
